.zenodo.json

```diff
@@ -0,0 +1,14 @@
+{
+    "title": "smesh",
+    "description": "tbd.",
```

Comment:
@simonechiocchetti Before merging, it would be good to add a short description of the "software".

For example, in Trixi.jl we have
> Adaptive high-order numerical simulations of hyperbolic PDEs in Julia

see
https://github.com/trixi-framework/Trixi.jl/blob/ba812be5ae6fb7170a85b9207253c8c89168ea76/.zenodo.json#L2

Note that this can be changed later of course, but not retroactively for already published versions